### PR TITLE
Signal insiders version in product name

### PR DESCRIPTION
**Pull request by jeanp413**
Related https://github.com/gitpod-io/gitpod/issues/5830

### CODE REVIEW

File: src/vs/gitpod/browser/workbench/workbench.ts [hunk 1]
```diff
@@ -40,6 +40,20 @@ interface ICredential {
...
+	instanceId: string
+	checkoutLocation: string
+	workspaceLocationFile?: string
+	workspaceLocationFolder?: string
+	userHome: string
+	gitpodHost: string
+	gitpodApi: { host: string }
+	workspaceContextUrl: string
+	workspaceClusterHost: string
+	ideAlias: string
```
Comment (jankeromnes) [inline src/vs/gitpod/browser/workbench/workbench.ts:54]: This field seems new -- I assume that's expected. 😁

File: src/vs/gitpod/browser/workbench/workbench.ts [hunk 2]
```diff
@@ -825,6 +826,9 @@ async function doStart(): Promise<IDisposable> {
 		urlCallbackProvider: new PollingURLCallbackProvider(),
 		credentialsProvider,
 		productConfiguration: {
+			nameShort: product.nameShort + (info.ideAlias === 'code-latest' ? ' - Insiders' : ''),
+			nameLong: product.nameLong + (info.ideAlias === 'code-latest' ? ' - Insiders' : ''),
+			version: product.version + (info.ideAlias === 'code-latest' ? '-insider' : ''),
```
Comment (jankeromnes) [inline src/vs/gitpod/browser/workbench/workbench.ts:831]: Maybe there should be an 's' in the version tag as well?

**Review by jankeromnes — COMMENTED** [review]
Code change looks good to me! (Modulo the two in-line questions)

However, I don't know how to test this. I guess the only way is via https://github.com/gitpod-io/gitpod/pull/5972 ?

**Review by jankeromnes — APPROVED** [review]
Tested via https://github.com/gitpod-io/gitpod/pull/5972 and works as expected! 🎉

Screenshot 2021-10-01 at 16 26 57

Screenshot 2021-10-01 at 16 22 52

Please feel free to address or discard the in-line questions and then merge. 🙏

File: src/vs/gitpod/browser/workbench/workbench.ts (hunk 2 above)
Comment (akosyakov) [reply src/vs/gitpod/browser/workbench/workbench.ts:831]: that's all right, seems to be aligned with how VS Code Desktop handles it

**Pull request merged**